### PR TITLE
[kr] Korean translation for lesson 3 sensors-and-actuators of 1-getting-started file

### PR DESCRIPTION
**Pull request by WZNT-KimHyeJin**
The contents of the lesson 3 sensors-and-actuators of 1-getting-started were translated into Korean.

- [x] README.kr.md
- [x] assignment.kr.md
- [x] pi-actuator.kr.md
- [x] pi-sensor.kr.md

### CODE REVIEW

**Comment by WZNT-KimHyeJin** [comment]
@microsoft-github-policy-service agree

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+✅ IoT 장치가 액추에이터가 처리할 수 있는 것보다 높은 전압을 전송하면 어떻게 된다고 생각하십니까?
+⛔️ 실험 해 보진 마십시오.
+
+### 펄스 폭 변조 (Pulse-Width Modulation)
+
+IoT 장치에서 아날로그 신호로 디지털 신호를 변환하는 또 다른 옵션은 펄스 폭 변조입니다. 이것은 마치 아날로그 신호인 것처럼 작동하는 많은 짧은 디지털 펄스를 보내는 것을 포함합니다.
+
+예를 들어 PWM을 사용하여 모터의 속도를 제어할 수 있습니다.
+
+5V 전원으로 모터를 제어한다고 가정해 보십시오. 모터에 짧은 펄스를 전송하여 전압을 200분의 2초(0.02초) 동안 하이(5V)로 전환합니다. 그 시간 동안 당신의 모터는 10분의 1 또는 36° 회전할 수 있다. 그런 다음 신호가 200분의 2초(0.02초) 동안 일시 중지되어 낮은 신호(0V)를 전송합니다. 각 ON/OFF 사이클은 0.04초 동안 지속됩니다. 그런 다음 주기가 반복됩니다.
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md:160]: - High(5V)
- 회전할 수 있다 -> 회전할 수 있습니다.

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+> 💁 일부 센서는 PWM을 사용하여 아날로그 신호를 디지털 신호로 변환하기도 합니다.
+
+> 🎓 [Wikipedia에서 펄스 폭 변조 검색](https://wikipedia.org/wiki/Pulse-width_modulation) 시 자세한 정보를 얻을 수 있습니다.
+
+### 디지털 액추에이터(actuator)
+
+디지털 센서와 같은 디지털 액추에이터는 고전압 또는 저전압에 의해 제어되는 두 가지 상태를 가지고 있거나 디지털 신호를 아날로그 신호로 변환할 수 있도록 DAC가 내장되어 있습니다.
+
+하나의 간단한 디지털 작동기는 LED이다. 장치가 1의 디지털 신호를 보내면 LED를 켜는 고전압이 전송됩니다. 0의 디지털 신호가 전송되면 전압이 0V로 떨어지고 LED가 꺼집니다.
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md:193]: - LED 이다 -> LED 입니다.

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+## 리뷰 & 추가 개별학습
+
+* 전기 및 회로에 대한 정보를 읽어보세요 [ThingLearn](http://thinglearn.jenlooper.com/curriculum/).
+* 다양한 유형의 온도 센서에 대한 자세한 내용은 [Seeed Studios Temperature Sensors guide](https://www.seeedstudio.com/blog/2019/10/14/temperature-sensors-for-arduino-projects/) 를 참조하세요
+* LED에 관한 내용은 [Wikipedia LED page](https://wikipedia.org/wiki/Light-emitting_diode) 에서 확인하세요
+
+## 과제
+
+[센서와 액추에이터에 ](assignment.md)
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/README.kr.md:222]: You have to change assignment's title.

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.kr.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+## 소개
+
+이 수업에서는 센서와 액추에이터에 대해 설명했습니다. IoT 개발 키트와 함께 사용할 수 있는 센서 하나와 액추에이터 하나를 알아봅시다. 여기에는 다음이 포함됩니다.
+
+* 무슨 일을 하나요?
+* 사용되는 전자 장치/하드웨어는 무엇이 있나요?
+* 아날로그 장치인가요 디지털장치인가요?
+* 입력 또는 측정의 단위 및 범위는 무엇입니까?
+
+## 루브릭(Rubric)
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.kr.md:12]: - 루브릭 -> 평가 기준

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.kr.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+| 기준 | 예시 | 적절성 | 개선 요구 사항 |
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.kr.md:14]: - | 모범 답안 | 적절함 | 개선이 필요함 |
unify the term

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**